### ISSUE TITLE
icu crate function deprecated?

### ISSUE THREAD

**Issue by sts10** [sts10/tidy#49]
At some point in a recent build process I saw the `icu` crate print a deprecation warning. I can't reproduce it just now, but we should think about upgrading Tidy's icu dependencies soon.

**Comment by sts10** [comment]
Here's the warning:
```text
warning: use of deprecated function `icu_testdata::unstable`: use `compiled_data`
  --> src/list_manipulations.rs:37:51
   |
37 |         Collator::try_new_unstable(&icu_testdata::unstable(), &locale.into(), options_l2).unwrap();
   |                                                   ^^^^^^^^
   |
   = note: `#[warn(deprecated)]` on by default

warning: `tidy` (lib) generated 1 warning
```

**Comment by sts10** [comment]
Fixed in b396c7b3c218ea082fe4da9e5aa6f27307551686
